Common/src/dual_grid_structure.cpp

```diff
@@ -659,8 +659,6 @@ CTurboVertex::CTurboVertex(unsigned long val_point, unsigned short val_nDim) : C
 
 CTurboVertex::~CTurboVertex() {
 
-	if (Normal != NULL) delete[] Normal;
```

Comment:
Why is that removed ? Is is deleted somewhere else ?

Reply:
Thanks for reading through my changes.  I like having a second set of eyes on the code.

`CTurboVertex` is derived from `CVertex`, so the destructor for `CTurboVertex` is called, then the destructor for `CVertex`.  Both destructors have these two lines (`delete [] Nodes` and `delete [] Normal`).  So I got a double-free when I tested a turbomachinery problem.

The simple solution is just to remove the lines from `CTurboVertex` and let the base class destructor handle the deallocation.